### PR TITLE
Updated Docker-image base to phusion/baseimage:0.11

### PR DESCRIPTION
**Pull request by mserranom**
Fixes https://github.com/overleaf/issues/issues/2043

### CODE REVIEW

File: Dockerfile-base [hunk 1]
```diff
@@ -24,7 +25,7 @@ RUN echo "selected_scheme scheme-basic" >> /install-tl-unx/texlive.profile; \
 RUN rm -r /install-tl-unx; \
 	rm install-tl-unx.tar.gz
 
-ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/texlive/2017/bin/x86_64-linux/
```
Comment (JuneKelly) [inline Dockerfile-base:27]: I think we'll want to stay on texlive 2017 until we have a fully-tested upgrade in production.

(we still ship texlive 2017 in prod, and we intend to upgrade it in the next few months)

**Review by JuneKelly — CHANGES_REQUESTED** [review]
Looks good, one change

**Review by JuneKelly — APPROVED** [review]

**Comment by mserranom** [comment]
On hold until Community/ServerPro images are revived and a decision over texlive version is final.

**Comment by mserranom** [comment]
Closing in favour of https://github.com/overleaf/docker-image/pull/107 that already includes the change.

**Pull request closed without merging**